package.json

```diff
@@ -51,6 +51,7 @@
     "express-winston": "4.0.2",
     "fetch-jsonp": "1.1.3",
     "file-saver": "2.0.2",
+    "formik": "^2.1.4",
```

Comment:
We use fixed dependencies. Also, can you commit the lockfile, CI is not happy if it's missed.

Reply:
The preview deployment is currently broken because of that. https://opencollective-frontend-idfrgkksew.now.sh/create/v2

Reply:
Please use fixed dependencies.